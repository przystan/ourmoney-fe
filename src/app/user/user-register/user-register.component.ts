import { Component, ChangeDetectionStrategy } from '@angular/core';

@Component({
  selector: 'app-user-register',
  template: `
    <p>
      user-register works!
    </p>
  `,
  styleUrls: ['./user-register.component.scss'],
  changeDetection: ChangeDetectionStrategy.OnPush
})
export class UserRegisterComponent {}
