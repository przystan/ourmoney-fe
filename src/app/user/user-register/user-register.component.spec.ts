import { Spectator, createComponentFactory } from '@ngneat/spectator';
import { UserRegisterComponent } from './user-register.component';

describe('UserRegisterComponent', () => {
  let spectator: Spectator<UserRegisterComponent>;

  const createComponent = createComponentFactory({
    component: UserRegisterComponent
  });

  beforeEach(() => {
    spectator = createComponent();
  });

  it('should match snapshot', () => {
    expect(spectator.fixture).toMatchSnapshot();
  });
});
