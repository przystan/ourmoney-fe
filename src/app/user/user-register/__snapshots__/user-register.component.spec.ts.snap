// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserRegisterComponent should match snapshot 1`] = `
<app-user-register>
  <p>
     user-register works! 
  </p>
</app-user-register>
`;
